libs/features/mas/test/hydrate.test.js

```diff
@@ -514,3 +550,127 @@ describe('processDescription', async () => {
         );
     });
 });
+
+describe('getTruncatedTextData', () => {
+  it('closes any open tags in truncated text', () => {
```

Comment:
can you add a test case for graceful handling in case an error happens inside `getTruncatedTextData`